docs/installation.md

```diff
@@ -0,0 +1,45 @@
+# Installation
+
+To install this application you should search for "Elastic Agent" in the GCP Marketplace and install it through the GCP Console.
+
+From the product listing you can choose the version of the application to install. We maintain `7.17.x` and `8.x` stack versions.
+
```

Comment:
Suggestion to add the following around what version to install => Review the [compatibility matrix](https://www.elastic.co/support/matrix#matrix_compatibility) for more details. Elastic recommends installing the latest version of the Elastic Agent that’s compatible with the version of your Elasticsearch deployment.

Reply:
added!